_posts/2021-07-28-how-to-write-blog-posts-for-strimzi-blog.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title:  "How to contribute blog posts for Strimzi blog"
+date: 2021-07-28
+author: jakub_scholz
+---
+
+Most of the posts on the Strimzi blog are written by Strimzi maintainers or regular contributors.
+But there is no rule preventing anyone else from writing and contributing blog posts.
+In fact, we are more than happy to publish your blog posts here.
+If you are interested in contributing a blog post, you can read more about how to do it ... in this blog post!
+
+<!--more-->
+
+## What topics to write about
+
+So, what can you write about?
+Of course, most readers of the Strimzi blog are interested in Strimzi and Apache Kafka.
+So that should be one of the main topics of the blog post.
+But there are plenty of things to write about.
+For example:
+
+* Your experience running Strimzi
+    * How you configured it in your environment
+    * What your architecture looks like
+    * How you made Strimzi and Apache Kafka work well for your use-case
+    * Strimzi and IoT or Edge computing
+* Tuning Strimzi configuration for different use-cases
+* Running Strimzi on different infrastructure
+    * Integration with different public or private clouds
+    * Using bare-metal infrastructure
+    * Using Strimzi on top of different Kubernetes distributions
+* Integration with other projects and tools
+    * Using Strimzi with different OAuth authorization servers
+    * How to connect to Strimzi from other applications
+    * How to configure different connectors
+    * Which UIs to use with Strimzi based Apache Kafka clusters
+    * Use of Open Policy Agent policies for authorization in Strimzi
+* Deep dives into Strimzi / Apache Kafka features
+* Other Strimzi sub-projects such as Kafka Bridge or our configuration providers
+* Tutorials and how to guides about different types of contributions (such as this blog post 😉)
+
+I'm sure you will be able to come up with many other topics which will be interesting for Strimzi users.
+You can also browse through the [older blog posts](https://strimzi.io/blog/) to find some inspiration and examples.
+
+While in Strimzi, we love open-source software, we also understand that not everything is open source.
+If you want to mention a commercial version of your project or service, it is not necessarily a blocker.
+Just keep in mind that the blog post should be also about Strimzi and Apache Kafka and not just a pure self-promotion.
+
+If you are not sure about the topic, you can always get in touch with us first.
+You can use the Slack channel, mailing list or DM us on Twitter to discuss the ideas.
+
+## How to contribute the blog post
```

Comment:
```suggestion
## How to contribute a blog post
```

Reply:
This section might be useful in the strimzi blog repo _readme_ or in the contributor guide